README.md

```diff
@@ -25,13 +25,18 @@ The table below lists the supported operating systems and Python versions requir
 | Red Hat* Enterprise Linux* 8, 64-bit                       | 3.6, 3.8                                             |
 | CentOS\* 7, 64-bit                                         | 3.6, 3.7, 3.8                                        |
 | macOS\* 10.15.x versions                                   | 3.6, 3.7, 3.8                                        |
-| Windows 10\*, 64-bit Pro, Enterprise or Education editions | 3.6, 3.7, 3.8                                        |
-| Windows Server\* 2016 or higher                            | 3.6, 3.7, 3.8                                        |
+| Windows 10\*, 64-bit Pro, Enterprise or Education editions | 3.6, 3.7, 3.8*                                        |
```

Comment:
Can you use a dagger (†) instead of asterisk (*) for the Windows Python 3.8 notice? Since we already use asterisk for the "property of others" notice.

Reply:
I can, but is there another option than that one? It doesn't look like a neutral sign to me. I remember thinking when I first came across that as a reference in a research paper I thought it meant the authors were deceased. If this is the best option it is okay though. Hopefully we can remove it soon.

Reply:
Good point. Maybe double asterisk? (**)

Reply:
Done.